andela/ah-jawans-frontend
# #166790061 user should be able to view notifications of user

#### What does this PR do?

Allow user to view notifications in real-time

#### Description of Task to be completed?

Users should be able to view notifications

#### How should this be manually tested?

Login to Authors havens the check your profile picture in the top right side of the home page and a bell icon after that.

After cloning the repo, cd into it, type npm run test. To run the application type npm start

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166790061

#### Screenshots (if appropriate)

![ezgif-4-5645a2435de4](https://user-images.githubusercontent.com/51748364/64256742-55fe1d00-cf24-11e9-8b6d-97884fb8dbd4.gif)
![ezgif-4-5846cd0dbb52](https://user-images.githubusercontent.com/51748364/64256922-ab3a2e80-cf24-11e9-94ba-6372f115da27.gif)
